pkgs/data/fonts/nerd-fonts/default.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  callPackage,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+let
+  releaseInfo = lib.trivial.importJSON ./manifests/release.json;
+  fontsInfo = (lib.trivial.importJSON ./manifests/fonts.json).fonts;
+  hashes = lib.trivial.importJSON ./manifests/hashes.json;
+
+  # Always aborts if finding nothing.
+  firstFound = pred: lib.lists.findFirst pred (throw "nothing found");
+
+  convertAttrName =
+    name:
+    let
+      lowerName = lib.strings.toLower name;
+    in
+    if builtins.match "[[:digit:]].*" lowerName != null then "_" + lowerName else lowerName;
+
+  convertLicense = callPackage ./convert-license.nix { };
+
+  makeNerdFont =
+    {
+      caskName,
+      description,
+      folderName,
+      licenseId,
+      patchedName,
+      version,
+      ...
+    }:
+    stdenvNoCC.mkDerivation {
+      pname = lib.strings.toLower caskName;
+      version =
+        if builtins.match "[[:digit:]].*" version != null then
+          version
+        else
+          "0-unstable-" + builtins.head (lib.strings.splitString "T" releaseInfo.published_at);
```

Comment:
This would break the consistency among versions of different fonts. And if the upstream does not want to specify some of the individual versions, why do we care about them?